.gitignore

```diff
@@ -1,5 +1,6 @@
 *.pyc
 src/tmp/
+src/Temp/

```

Comment:
@Raghu-Srungavarapu What is this folder needed for?

Reply:
Actually, when I think of it again, This doesn't seem like required line,
Previously I used this to stores the output files from top counts.  However, the path to this folder is user-defined. I am going to change my path to local and delete this row.